src/modules/commander/Commander.hpp

```diff
@@ -132,7 +132,7 @@ class Commander : public ModuleBase<Commander>, public ModuleParams
 	void battery_status_check();
 
 	bool check_posvel_validity(const bool data_valid, const float data_accuracy, const float required_accuracy,
-				   const hrt_abstime &data_timestamp_us, hrt_abstime *last_fail_time_us, hrt_abstime *probation_time_us,
+				   const hrt_abstime &data_timestamp_us, hrt_abstime *last_fail_time_us,
```

Comment:
```suggestion
				   const hrt_abstime &data_timestamp_us, hrt_abstime &last_fail_time_us,
```
Since you're changing that line already, could you switch `last_fail_time_us` to a reference instead of pointer?

Reply:
Good point, I'll follow up. 👍